lib/runner.js

```diff
@@ -46,6 +46,7 @@ function Runner(options) {
 
 	this.results = [];
 	this.tests = new TestCollection();
+	this.hasBegunRunning = false;
```

Comment:
I didn't name it is running because it isn't reset to true after the tests are done running.